46.md

```diff
@@ -184,18 +185,6 @@ Clients should display (in a popup or new tab) the URL from the `error` field an
 
 ![signing-example-with-auth-challenge](https://i.nostr.build/W3aj.png)
 
-## Remote Signer Commands
-
-Remote signers might support additional commands when communicating directly with it. These commands follow the same flow as noted above, the only difference is that when the client sends a request event, the `p`-tag is the pubkey of the remote signer itself and the `content` payload is encrypted to the same remote signer pubkey.
-
-### Methods/Commands
-
-Each of the following are methods that the client sends to the remote signer.
-
-| Command          | Params                                     | Result                               |
-| ---------------- | ------------------------------------------ | ------------------------------------ |
-| `create_account` | `[<username>, <domain>, <optional_email>, <optional_requested_permissions>]` | `<newly_created_remote_user_pubkey>` |
-
```

Comment:
Why remove this? From what I've seen it's pretty standard/necessary for a complete flow.

Reply:
I have moved this method to the other table. These other considerations about "additional commands" felt unnecessary and confusing.